.gitignore

```diff
@@ -31,5 +31,7 @@ node_modules/
 
 centrifuge-chain/
 
-contracts/
+./contracts/
 abi/
+
+!on-chain/evm-contracts/test/bridge/
```

Comment:
I think `bridge` can be replaced with `chainbridge` above (we renamed the Go executable), which I think removes the need for this